aziotctl/aziotctl-common/src/restart.rs

```diff
@@ -0,0 +1,59 @@
+use std::io::{self, Write};
+use std::process::Command;
+
+use crate::ServiceDefinition;
+
+pub fn restart(services: &[&ServiceDefinition]) {
+    // stop all services
+    for ServiceDefinition {
+        service,
+        sockets: _,
+    } in services
+    {
+        print!("Stopping {}...", service);
+        let result = Command::new("systemctl")
+            .args(&["stop", service])
+            .output()
+            .unwrap();
+
+        if result.status.success() {
+            println!("Stopped!");
+        } else {
+            println!("\nError stopping {}", service);
+            io::stdout().write_all(&result.stdout).unwrap();
+            io::stderr().write_all(&result.stderr).unwrap();
```

Comment:
It's fine to leave these unwraps, given that `println!` does pretty much the same thing under the hood anyways.